rpm/src/test/java/org/eclipse/packager/rpm/signature/RpmFileSignatureProcessorTest.java

```diff
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: EPL-2.0
  *
  * Contributors:
- *   mat1e, Groupe EDF - initial API and implementation
- *   Jens Reimann, Red Hat, Inc
+ * mat1e, Groupe EDF - initial API and implementation
```

Comment:
Don't change existing contributor lines, but add yourself as one.

Reply:
This must have been some `spotless apply` removing the whitespace, sorry.